packages/app-desktop/gui/NoteEditor/utils/useFormNote.ts

```diff
@@ -97,7 +97,15 @@ const useRefreshFormNoteOnChange = (formNoteRef: RefObject<FormNote>, editorId:
 
 			await initNoteState(n, false);
 			if (event.cancelled) return;
-			setFormNoteRefreshScheduled(0);
+			setFormNoteRefreshScheduled(oldValue => {
```

Comment:
See [ReactJS: Setting state based on the previous state](https://react.dev/reference/react/useState#updating-state-based-on-the-previous-state).